iphone/Classes/TiUIWebView.m

```diff
@@ -775,10 +775,9 @@ - (void)webViewDidFinishLoad:(UIWebView *)webView
         }
     }
     
-    // Disable user selection and the attached callout
-    BOOL disableSelection = [TiUtils boolValue:[[self proxy] valueForKey:@"disableSelection"] def:NO];
+    // Disable the context menu when selecting text
+    BOOL disableSelection = [TiUtils boolValue:[[self proxy] valueForKey:@"disableContextMenu"] def:NO];
```

Comment:
Agree. The Travis job is running for 2 days now, hopefully it's re-triggered correctly after the commit.